lib/components/Widgets/Charts/ChartContainer.tsx

```diff
@@ -1,14 +1,8 @@
 import { forwardRef, ReactNode } from "react"
 import { WidgetContainer, WidgetContainerProps } from "../WidgetContainer"
-import ChartCounter from "./chart-counter"
 
 export type ChartContainerPropsBase = WidgetContainerProps & {
-  summaries?: Array<{
-    label: string
-    value: number
-    prefixUnit?: string
-    postfixUnit?: string
-  }>
+  summaries?: WidgetContainerProps["summaries"]
```

Comment:
This shouldn't be needed as the `ChartContainerProps` now are just `WidgetContainerProps`. Have you tried just doing `ChartContainerPropsBase = WidgetContainerProps`?

Reply:
Ah, right! Done 🫶